RunScripts/en-de_ablation_pipeline.sh

```diff
@@ -35,19 +35,21 @@ python -u -m EC_finetune --config Configs/${EC_CONFIG}.yml \
 # de2en=$(./Tools/bleu.sh ${OUTPUT_DIR}/de-en.de.val.en ${OUTPUT_DIR}/de-en.en.val 13a)
 # echo 'en to de score: '"$en2de"'; de to en score: '"$de2en"
 
-# Do rest of backtranslation
-python -u BackTranslation/backtranslate.py --config Configs/${BT_SECONDARY_CONFIG}.yml \
-    --seed_override 2 \
-    --model_dir_override Output/en-de_pipeline/ec_${EX_ABBR} \
-    --output_dir_override ${OUTPUT_DIR}
+# # Do rest of backtranslation
+# python -u BackTranslation/backtranslate.py --config Configs/${BT_SECONDARY_CONFIG}.yml \
+#     --seed_override 2 \
+#     --model_dir_override Output/en-de_pipeline/ec_${EX_ABBR} \
+#     --output_dir_override ${OUTPUT_DIR}
 
 # Do test
-cp /projects/unmt/communication-translation/Data/translation_references/de-en.* ${OUTPUT_DIR}
-python -u BackTranslation/translate.py --config Configs/en2de_translate.yml \
-    --output_dir ${OUTPUT_DIR} \
-    --model_path ${OUTPUT_DIR}/best_bleu
-python -u BackTranslation/translate.py --config Configs/de2en_translate.yml \
-    --output_dir ${OUTPUT_DIR} \
-    --model_path ${OUTPUT_DIR}/best_bleu
+# cp /projects/unmt/communication-translation/Data/translation_references/de-en.* ${OUTPUT_DIR}
+# python -u BackTranslation/translate.py --config Configs/test_en2de_translate.yml \
+#     --output_dir ${OUTPUT_DIR} \
+#     --model_path ${OUTPUT_DIR}/best_bleu
+# python -u BackTranslation/translate.py --config Configs/test_de2en_translate.yml \
+#     --output_dir ${OUTPUT_DIR} \
+#     --model_path ${OUTPUT_DIR}/best_bleu
+
```

Comment:
Why is everything commented out?

Reply:
Because we are not doing the test here.

Reply:
But this is a lot more than testing that's commented out; pretty much everything is